doc/api/next_api_changes/behavior/19438-TAC.rst

```diff
@@ -0,0 +1,49 @@
+``plt.subplot`` re-selection without keyword arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The purpose of `.pyplot.subplot` is to facilitate creating and
+re-selecting Axes in a Figure when working in the implicit pyplot API.
+When used to create Axes it is possible to select the projection
+(e.g. polar, 3D, or various cartographic projections) as well as to pass
+additional keyword arguments through to the Axes-subclass that is created.
+
+The first time `.pyplot.subplot` is called for a given position in the
+Axes grid it is clear that the correct behavior is to create and
+return an Axes at the given position with the passed arguments
+(defaulting to a rectilinear Axes).  However, on subsequent calls to
+`.pyplot.subplot` the correct behavior is more subtle.  If an Axes exists
+at the requested location and has equivalent parameters the existing Axes
+should be returned, but if the parameters are different the old Axes is removed
+from the Figure and a new Axes is created and returned.
+
+Due to an implementation detail, it was previously the case that for all
```

Comment:
Is this implementation detail what was introduced by my changes in #19153? If so, we need not say that it was "previously the case" because those changes have never been in a release yet.

Reply:
no, this is a bug that goes back to 2.2.

Reply:
* atleast 2.2

Reply:
but this text is missing a crucial half-sentence ....

Reply:
Got it. Thanks!